### DISCUSSION TITLE
Programming via wifi

### DISCUSSION THREAD

**Discussion by florisniehaus** [BIPES/BIPES#148] [General]
I am new to Bipes, I can program via serial port, how do I program ESP2866 via wifi

**Comment by rafaelaroca** [comment]
Hello! You sure can! You just have to enable webrepl on the ESP8266 and then connect the board to the network. After that, you can program it over wifi!

1. Use the block to connect to a wifi network. 
2. Only once, type the  command to enable webrepl

import webrepl_setup

Here are the details, if you want to read about:

http://bipes.net.br/docs/get-started/tech-details.html

After that, simply go to BIPES and select connect using the network, and type the board address!

Please, let us know if everything works fine!

best regards

**Reply by florisniehaus** [reply]
Thanks, I did manage to connect through wifi. You truly create an excellent product, Keep up the good work, this will pay off!!!
